portfolio/src/main/webapp/style.css

```diff
@@ -2,8 +2,37 @@
   margin-left: auto;
   margin-right: auto;
   width: 650px;
+  font-family: 'Roboto', sans-serif;
+  text-align: center;
 }
 
-#greeting-container {
-  margin-top: 20px;
+#name {
+    font-family: 'Inconsolata', monospace;
+    font-weight: 200;
+    font-size: 32px;
 }
```

Comment:
This whole file should be alphabetized with declaration order: 
https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

Also, shorthand things like "font":

https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties